oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/IndexInfoServiceImpl.java

```diff
@@ -62,11 +63,21 @@ public IndexInfoServiceImpl(NodeStore nodeStore, IndexPathService indexPathServi
 
     @Override
     public Iterable<IndexInfo> getAllIndexInfo() {
+        HashSet<String> allIndexes = new HashSet<>();
+        indexPathService.getIndexPaths().forEach(allIndexes::add);
+        HashSet<String> activeIndexes = new HashSet<>();
+        if (indexPathService.getMountInfoProvider().hasNonDefaultMounts()) {
+            IndexName.filterReplacedIndexes(allIndexes, nodeStore.getRoot(), true).forEach(activeIndexes::add);
```

Comment:
```suggestion
            activeIndexes.addAll(IndexName.filterReplacedIndexes(allIndexes, nodeStore.getRoot(), true));
```

Reply:
Right (I tried using addAll first but it didn't compile, so I switched to this... but I must have made some mistake, because now it does compile)